toolkit/tools/imagecustomizer/docs/iso.md

```diff
@@ -90,7 +90,8 @@ iso:
     cloud-init-data/meta-data: /cloud-init-data/meta-data
 
   kernelCommandLine:
```

Comment:
You should also update `configuration.md` changing `### extraCommandLine [string]` to `### extraCommandLine [string[]]`